pkgs/development/haskell-modules/hie-packages.nix

```diff
@@ -0,0 +1,482 @@
+{ pkgs, stdenv, callPackage }: self:
+let src = pkgs.fetchFromGitHub
+      { owner = "haskell";
+        repo = "haskell-ide-engine";
+        rev = "e9059d683e4a1a73cc16d3ddae821428b6f2d7a6";
+        sha256 = "1n4wwd5baqx0wjhjmg5lf90cyg61czw31wqniwm16fsn993z4rix";
+      };
+    cabal-helper-src = pkgs.fetchgit
+      { url = "https://gitlab.com/dxld/cabal-helper.git";
+        rev = "4bfc6b916fcc696a5d82e7cd35713d6eabcb0533";
+        sha256 = "1a8231as0wdvi0q73ha9lc0qrx23kmcwf910qaicvmdar5p2b15m";
+      };
+    ghc-dump-tree-src = pkgs.fetchgit
+      { url = "https://gitlab.com/alanz/ghc-dump-tree.git";
+        rev = "50f8b28fda675cca4df53909667c740120060c49";
+        sha256 = "0v3r81apdqp91sv7avy7f0s3im9icrakkggw8q5b7h0h4js6irqj";
+      };
+    ghc-mod-src = pkgs.fetchFromGitHub
+      { owner = "wz1000";
+        repo = "ghc-mod";
+        rev = "03c91ea53b6389e7a1fcf4e471171aa3d6c8de41";
+        sha256 = "11iic93klsh5izp8v4mhl7vnnlib821cfhdymlpg4drx7zbm9il6";
+      };
+    HaRe-src = pkgs.fetchgit
+      { url = "https://gitlab.com/alanz/HaRe.git";
+        rev = "e325975450ce89d790ed3f92de3ef675967d9538";
+        sha256 = "0z7r3l4j5a1brz7zb2rgd985m58rs0ki2p59y1l9i46fcy8r9y4g";
+      };
+    cabal-helper = self.cabal-helper_hie;
+    haddock-library = self.haddock-library_1_4_4;
+    ghc-dump-tree = self.ghc-dump-tree_hie;
+    ghc-mod = self.ghc-mod_hie;
+    HaRe = self.HaRe_hie;
+in
+  { # This isn't really hie-specific, should probably go in cabal2nix
+    haddock-library_1_4_4 = callPackage
```

Comment:
@peti What's the right way to add this to hackage-packages.nix?

Reply:
Just add it to the `extra-packages` section in `pkgs/development/haskell-modules/configuration-hackage2nix.yaml`. Then it will appear in the main package set within a couple of days (i.e. whenever I think of merging `haskell-updates` into `master`).